### ISSUE TITLE
Change plugin id to reflect the move on mattermost github group

### ISSUE THREAD

**Issue by manland** [mattermost/mattermost-plugin-gitlab#67]

**Comment by hanzei** [comment]
Keep in mind that this will be a breaking change for all existing installations.

**Comment by manland** [comment]
Yes we need to say it's breaking and delete old `manland...` and report conf to new `github...`!? Changelog is enough not?

**Comment by manland** [comment]
O yes and data in kvstore will be forgotten... O sh*t... What to do ?

**Comment by hanzei** [comment]
The KVstore is the big problem.

> O sh*t... What to do ?

:cry:

**Comment by hanzei** [comment]
If you have idea how to support a change of plugin id's in MM, feel free to propose it in the extensions channel.

**Comment by jasonblais** [comment]
I posted here to start the conversation: https://community-release.mattermost.com/core/pl/g9msf4a4bpy9zgrk1swbw8ocfe
